### DISCUSSION TITLE
kafkauser not working after upgrade to 0.32

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#8143] [General]
Hi I am seeing issue that user is getting created but unfortunate I am not able to use the user to produce or consume.   please can you validate if the manifest using is okay 

I have upgraded from 0.28 to 0,32. and I am creating a new user -with below manifest

apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaUser
metadata:
	name: 907100
labels:
	strimzi.io/cluster: test
spec:
authentication:
	type: tls-external
authorization:
	type: simple
acls:
  - resource:
      type: group
      name: "*"
      patternType: literal
    operations:
      - Read
      - Write
      - Create
      - Delete
      - Alter
      - Describe
      - ClusterAction
      - AlterConfigs
      - DescribeConfigs
      - IdempotentWrite
      - All
    host: "*"

  - resource:
      type: group
      name: "*"
      patternType: literal
    operations:
      - Read
      - Write
      - Create
      - Delete
      - Alter
      - Describe
      - ClusterAction
      - AlterConfigs
      - DescribeConfigs
      - IdempotentWrite
      - All
    host: "*"
  - resource:
      type: transactionalId
      name: "*"
      patternType: literal
    operations:
      - Read
      - Write
      - Create
      - Delete
      - Alter
      - Describe
      - ClusterAction
      - AlterConfigs
      - DescribeConfigs
      - IdempotentWrite
      - All
    host: "*"

**Comment by sreejesh123** [comment]
Only difference I can see from the example given for 0.32 is I am using name: "*". For topic name , is this not allowed any more ? 

@scholzj please can you comment on this  ?

**Comment by scholzj** [comment]
Can you please properly format it? Without that, it is not readable and clear how your YAML looks like.

**Comment by sreejesh-radhakrishnan-db** [comment]
will try to format, but what ever I am trying the PREVIEW is showing non-formated way.    But one QQ

0.32  do we support both OPEARATION:  and OPERATIONS:   as in the crd I can see both.

**Reply by scholzj** [reply]
`operation` is still supported. Buz it is deprecated and `operations` should be used in the long term.

**Comment by sreejesh-radhakrishnan-db** [comment]
Its working.  sorry for wasting everyone time.  

the error I was getting was "broker0.kafka.gpds.dev.gcp.db.com:9093 (id: 0 rack: europe-west3-a): Timed out waiting for a node assignment. Call: listConsumerGroups"  which I inferred as  kafauser issue.   but actually somehow KAFKA broker svc couldn't find the actual pod for kafka-0 for some reason. I reinstalled the kafka and its all working now.  weird but its all working.
